README.md

```diff
@@ -80,6 +111,8 @@ download a pre-build version).
 ./meteor --help
 ```
 
```

Comment:
I agree adding an alias etc. makes things more comfortable. My approach was to behave like a first time contributor who finds a simple issue he/she wants to fix. Personally I think we should add advanced methods, like adding an alias, into https://github.com/meteor/meteor/blob/devel/Contributing.md for example.